src/main/resources/db/changelog/changes/01-create-book-table.yaml

```
@@ -0,0 +1,46 @@
+databaseChangeLog:
+  - changeSet:
+      id: createBooksTable
+      author: your_name
```

You can place here your name or nickname, no need to senseless copying. 
```suggestion
      author: your_name
```